socrates/test/members/members_test.js

```diff
@@ -123,12 +135,77 @@ describe('SoCraTes members application', function () {
     });
 
     it('allows a Softwerkskammer member to edit his page', function (done) {
+      sinon.stub(subscriberstore, 'getSubscriber', function (nickname, callback) { callback(null, softwerkskammerSubscriber); });
+
       appWithSoftwerkskammerMember
         .get('/edit')
         .expect(200)
         .expect(/Here you find the information from your Softwerkskammer account that is used by SoCraTes\./, done);
     });
 
+    it('does not allow an unsubscribed member to enter a roommate', function (done) {
```

Comment:
Du nimmst den Body des tests und machst ne Funktion draus: 

body ist 
 sinon.stub(subscriberstore, 'getSubscriber', function (nickname, callback) { callback(null, socratesSubscriber); });
+
+      appWithSocratesMember
+        .get('/edit')
+        .expect(200)
+        .expect(/Home Address/)
+        .end(function (err, res) {
+          expect(res.text).to.not.contain('Who do you want to share your room with');
+          done(err);
+        });

"Parameter" ist socrates.resources.single._registeredMembers = [{memberId: 'memberId2'}]; bzw der andere raum. Get it?